common/src/device/provisioning/transport/DeviceRegistrationResult.cs

```diff
@@ -14,128 +14,64 @@ namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Models
-        /// </summary>
-        partial void CustomInit();
-
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "tpm")]
         public TpmRegistrationResult Tpm { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "x509")]
         public X509RegistrationResult X509 { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "symmetricKey")]
         public SymmetricKeyRegistrationResult SymmetricKey { get; set; }
 
-
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "registrationId")]
         public string RegistrationId { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "createdDateTimeUtc")]
         public DateTime? CreatedDateTimeUtc { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "assignedHub")]
         public string AssignedHub { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "deviceId")]
         public string DeviceId { get; set; }
 
         /// <summary>
-        /// Gets or sets possible values include: 'unassigned', 'assigning',
-        /// 'assigned', 'failed', 'disabled'
+        /// The status of the operation.
+        /// Possible values include: 'unassigned', 'assigning', 'assigned', 'failed', 'disabled'
         /// </summary>
         [JsonProperty(PropertyName = "status")]
         public string Status { get; set; }
 
         /// <summary>
+        /// The substatus of the operation.
         /// Possible values include: 'initialAssignment', 'deviceDataMigrated', 'deviceDataReset'
         /// </summary>
         [JsonProperty(PropertyName = "substatus")]
         public string Substatus { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "generationId")]
         public string GenerationId { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "lastUpdatedDateTimeUtc")]
         public DateTime? LastUpdatedDateTimeUtc { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "errorCode")]
         public int? ErrorCode { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "errorMessage")]
         public string ErrorMessage { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "etag")]
         public string Etag { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "payload")]
         public JRaw Payload { get; set; }
+
+        /// <summary>
+        /// The PEM encoded operational client certificate that was returned by the CA. This operational certificate was used by DPS to register the enrollment with IoT Hub.
```

Comment:
```suggestion
        /// The PEM encoded operational client certificate that was returned by the certificate authority. This operational certificate was used by DPS to register the enrollment with IoT Hub.
```

I'd avoid using acronyms when possible since users may not be familiar with them.